package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "wasm-re2",
-  "version": "1.1.0",
+  "name": "@dashevo/re2",
```

Comment:
Let's save original wasm prefix in case if we gonna use other binding or original lib in our organization